src/kernel/arch/x86/arch.zig

```diff
@@ -100,4 +125,73 @@ export nakedcc fn start_higher_half() noreturn {
     while (true) {}
 }
 
+///
+/// Initialise the architecture
+///
+/// Arguments:
+///
+/// Global changes:
+///
+/// Return:
+///
```

Comment:
We could probably do without empty sections (arguments, global changes and return).

Reply:
Can remove empty sections